packages/voting/graphql/schema-types.js

```diff
@@ -146,6 +146,7 @@ type Candidacy {
 
   yearOfBirth: Int
   city: String
+  credential: Credential
```

Comment:
Resolving the credential does not work yet when queried via `election.candidacies` or `me.candidacies`:
<img width="838" alt="Screenshot 2021-08-29 at 00 14 07" src="https://user-images.githubusercontent.com/410211/131232295-563867f2-426c-4248-b402-4cac119ebcd7.png">

Reply:
Right. Done with https://github.com/orbiting/backends/pull/633/commits/f20e06fbbc919153d18b87644de06b4c825cb7e6